### ISSUE TITLE
Harvester not destroying custom crops

### ISSUE THREAD

**Issue by Teenkung123** [xenondevs/Nova#236]
### Agreements

- [X] I am running the latest version of Nova.
- [X] I am running the latest version of my server software.
- [X] I have checked for similar issues.
- [X] I am using an official build of Nova.

### Before creating a bug report, please validate that the issue you're encountering is not caused by another plugin (Remove all other plugins from the server). If it is, please name the plugin that causes the issue.

This issue is about a specific plugin incompatibility.

### Plugin Version, Server Software, Server Version

Nova V12.16, Purpur-1926 1.19.3, ItemsAdder 3.4.0 betar-11 

### Nova Addons and Versions

Machines: 0.3.2 Logistics: 0.2.3 Jetpack: 0.1.7

### Expected behavior

after placing down the Harvester and powered it, it should harvest the custom crop and destroy the custom crop 

### Observed/Actual behavior

Harvester actually Harvest the custom crop but it does not destroy the custom crop so It can dupe the crop

### Steps to reproduce

1. Place the harvester and set it up in the layout like this
![2023-03-08_17 27 20](https://user-images.githubusercontent.com/51858378/223688792-0ab4e8a1-cfeb-4691-99a6-9a52dfb5c623.png)
2. Plant the custom crop and use bonemeal to make it fully growth
3. Powered then Harvester, Insert the tool and let it run
4. You can see that it harvesting the custom crop but it does not destroy the crop 
![2023-03-08_17 29 03](https://user-images.githubusercontent.com/51858378/223689155-8ee97046-6ab1-49fa-8b67-00631b0c600d.png)

### Full server log

https://pastebin.com/3eAr5qhR

**Comment by NichtStudioCode** [comment]
I cannot reproduce this with ItemsAdder 3.4.1-r6, Nova 0.13-alpha.5, Machines `0.4-RC.4

**Comment by Teenkung123** [comment]
The problem is solved, but I got another problem related to custom crops.

My plugin hooked into Nova API to listen to TileEntityBlockBreakEvent, but it's turned out that Nova calls an unnecessary block break event when a harvester tries to harvest the crop without having the tool to do it. So, it just calls block break event for every block it tries to harvest but not actually breaking that block.

And another problem I think is related to the problem above, after Nova called an unnecessary TileEntityBlockBreakEvent, if the block it tried to harvest is a custom crop, it will turn into a normal crop like the picture below.

![2023-05-03_01 35 06](https://user-images.githubusercontent.com/51858378/235755141-92645421-7076-4045-b758-d1e0fb023658.png)
[At the action bar, you will see +4 Farming XP, but the harvester doesn't have the tools to do the work. Yet, Nova still called a TileEntityBlockBreakEvent.]

[In WAILA, it actually is strawberry seeds from ItemsAdder, but after Nova called a TileEntityBlockBreakEvent, it got replaced with normal wheat.]
